java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -2948,9 +2948,6 @@ public void bzmpop_timeout_check(BaseClient client) {
                         ? RedisClient.CreateClient(commonClientConfig().build()).get()
                         : RedisClusterClient.CreateClient(commonClusterClientConfig().build()).get()) {
 
-            // ensure that commands doesn't time out even if timeout > request timeout
-            assertNull(testClient.bzmpop(new String[] {key}, MAX, 1).get());
```

Comment:
why?

Reply:
I don't know... must be a merging conflict.  Will revert.

Reply:
Oh - I was showing something to Chloe on this test.